### ISSUE TITLE
Track (conductor) routing on component move

### ISSUE THREAD

**Issue by krotow** [Proto-App/Proto-Android#343]
In Proto app tracks (conductors) between components are made from loose straight lines. When I want to move component elsewhere, tracks doesn't move behind component, but lean under distance-dependent angle instead. The worst - it is not possible to correct then into straight lines anymore, because line connections aren't movable. The only way is to delete slanted tracks and draw them again correctly. It is very bad. Even KiCad did that right. For example between similar apps track autorouting is great selling point for EveryCircuit (they took that in extreme though without any possibility to pin tracks in place where that is required).

In short - can you do something to make tracks correcting themselves when connected components are moved? It may not happen instantly - there can be a screen button or something else for that. The best approach would be track autorouting in similar way what it happens in EveryCircuits, but with a possibility to pin some tracks in place to avoid weird track configurations.

**Comment by Proto-App** [comment]
Hello @krotow 
Yes, autorouting is on our TODO list.
